### PR TITLE
test(cypress): add tests for forking and updating projects

### LINKED ISSUE #2853: Cypress tests for project upgrades
Tests project upgrades by doing the following:

- [ ] create a `renku-ui-tests` group on GitLab containing a few projects useful for testing
- [ ] add an outdated but updatable project (v8) to the group
    -  ℹ️ v8 support will be phased out in the next renku-core release, and in the following one v10 will be introduced
- [ ] fork the updatable project, try to update the project, verify it works. 
- [ ] go to Overview --> Commits and verify there is a new update commit

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
Add tests to:
- Fork a project.
- Update a project from the previous version and verify the commits are pushed correctly.
- Assess we can't update very old projects.

This includes a few optimizations to the public project tests to improve stability.

/deploy #notest #cypress #persist
fix #2853

### CODE REVIEW

File: cypress-tests/cypress/e2e/updateProjects.cy.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+
+const username = Cypress.env("TEST_USERNAME");
+
+const projects = {
+  shouldFork: true,
+  namespace: "renku-ui-tests",
+  v7: "renku-project-v7",
+  v8: "renku-project-v8",
+}
```
Comment (ciyer) [inline cypress-tests/cypress/e2e/updateProjects.cy.ts:12]: I think a comment after this explaining how to modify the configuration to aid local debugging would be good.

**Review by ciyer — CHANGES_REQUESTED** [review]
This look really good! I think a comment explaining what do to for local debugging would be nice, but I think this can be merged otherwise. 

One note -- Locally, I had to run the new updateProjects test a few times before they passed. I didn't make any changes, just re-ran them. Maybe something to keep an eye on.

File: cypress-tests/cypress/e2e/updateProjects.cy.ts (hunk 1 above)
Comment (lorenzo-cavazzi) [reply cypress-tests/cypress/e2e/updateProjects.cy.ts:12]: Goog point! I added a comment

**Comment by lorenzo-cavazzi** [comment]
> One note -- Locally, I had to run the new updateProjects test a few times before they passed. I didn't make any changes, just re-ran them. Maybe something to keep an eye on.

Ha, I didn't have this problem. I tried them a couple of times once they were ready and they worked fine.
If you had the same error multiple times, please feel free to share it and I'm happy to improve the stability before we merge the PR. Having flaky tests won't be good anyway

**Comment by ciyer** [comment]
> > One note -- Locally, I had to run the new updateProjects test a few times before they passed. I didn't make any changes, just re-ran them. Maybe something to keep an eye on.
> 
> Ha, I didn't have this problem. I tried them a couple of times once they were ready and they worked fine. If you had the same error multiple times, please feel free to share it and I'm happy to improve the stability before we merge the PR. Having flaky tests won't be good anyway

I think we can merge then. It's possible that the flakiness on my computer was caused by something else.

**Review by ciyer — APPROVED** [review]
🎉

**Pull request merged**